### ISSUE TITLE
Refactor Child Visit to use rules engine

### ISSUE THREAD

**Issue by rkodev** [OpenSRP/opensrp-client-chw#298]
We need to refactor the child visit code to start using rules engine

**Comment by cagulas** [comment]
Once this issue is complete, https://github.com/OpenSRP/opensrp-client-chw/issues/143 becomes unblocked.

**Comment by cafootitt** [comment]
Tested all child health services for Liberia. One issue for vaccines, which I'll create a new issue.
